imx8m/edm_imx8m/BoardConfig.mk

```diff
@@ -139,6 +139,10 @@ ifeq ($(DISPLAY_TARGET),DISP_HDMI)
 BOARD_KERNEL_CMDLINE := init=/init androidboot.hwrotation=0 androidboot.gui_resolution=1080p androidboot.console=ttymxc0 androidboot.hardware=freescale androidboot.fbTileSupport=enable cma=$(CMASIZE) androidboot.primary_display=imx-drm firmware_class.path=/vendor/firmware transparent_hugepage=never
 else ifeq ($(DISPLAY_TARGET),DISP_MIPI_ILI9881C)
 BOARD_KERNEL_CMDLINE := init=/init androidboot.hwrotation=90 androidboot.console=ttymxc0 androidboot.hardware=freescale androidboot.fbTileSupport=enable cma=$(CMASIZE) androidboot.primary_display=imx-drm firmware_class.path=/vendor/firmware transparent_hugepage=never
+else ifeq ($(DISPLAY_TARGET),DISP_MIPI_G101UAN02)
```

Comment:
please write the bootargs together for 5", 8", and 10" lcd, they're use the same bootargs, isn't it?